### ISSUE TITLE
mrview: Overlay image switch causes resize

### ISSUE THREAD

**Issue by Lestropie** [MRtrix3/mrtrix3#1302]
When the Overlay tool is docked to the main window, and you have altered the relative size of the toolbar with respect to the GL window, if you then click on any of the check boxes that switch which overlay images are visible and which are not, the relative sizees of the GL window & toolbar are reset. Does not occur if the overlay toolbar is in floating mode.

May well happen with other tools in other cases; haven't done exhaustive testing. But it's annoying when you set up the GL window in a particular way in order to take a series of screenshots, and the window then gets resized as you make modifications to toolbar settings with the intention of taking multiple screenshots with the same size & FoV.

**Comment by thijsdhollander** [comment]
> But it's annoying when you set up the GL window in a particular way in order to take a series of screenshots, and the window then gets resized as you make modifications to toolbar settings with the intention of taking multiple screenshots with the same size & FoV.

Yes, I can relate to that.  However, this is also the case (and much harder to foresee) when opening a new tool during this process.  Most of the time when I'm taking a long series of such screenshots that have to be carefully crafted/aligned, I open all tools beforehand and undock them (and optionally move them to my second screen, so I can go full screen (resolution) for the actual screenshots).

I guess we can keep this effect to a minimum at least on a per-tool basis though.  That said, about this particular case:

> When the Overlay tool is docked to the main window, and you have altered the relative size of the toolbar with respect to the GL window, if you then click on any of the check boxes that switch which overlay images are visible and which are not, the relative sizees of the GL window & toolbar are reset.

I can't reproduce that.  I can happily switch visibility of multiple individual overlays after having resized the docked overlay tool without it resetting its relative size...
